src/NumberGuessingGame.js

```diff
@@ -1,77 +1,79 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import GuessControl from "./GuessControl";
 import GuessMessage from "./GuessMessage";
 import GameOver from "./GameOver";
 
-/**
- *
- * Returns a random integer number from 1-100 inclusive
- */
 function getRandomNumber() {
   return Math.floor(Math.random() * 100) + 1;
 }
 
-const MAX_ATTEMPTS = 5;
+const MAX_ATTEMPTS = 4;
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
 
-    this.state = {
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    };
+export default function NumberGuessingGame  () {
+  const [currentGuess, setCurrentGuess] = useState("");
+  const [numberOfGuesses, setNumberOfGuesses] = useState(0);
```

Comment:
Because of this, the number you're guessing always starts with zero. Take a look at the original code. See how they convert a string to a number so a user can actually blank/empty a the number guessing field

Reply:
Hi Kasozi, can you clarify a bit on this? You mean I should change this` const [numberOfGuesses, setNumberOfGuesses] = useState(0)`? Isn't it should always start at 0 whenever user start guessing the game?? Like it's always the first time they guess, and whenever they make another guess it would add +1 more? In the original code I also see them do      ` numberOfGuesses: 0` so I am a bit confused what should I change excactly in this case?